examples/ex_coreg/main.cpp

```diff
@@ -0,0 +1,244 @@
+        bScale = true;
+    }
+
+    float fTol = parser.value(tolOption).toFloat();
+    float fMaxDist = parser.value(distOption).toFloat();
+    int iMaxIter = parser.value(iterOption).toInt();
+
+    // read Trans
+    FiffCoordTrans transHeadMriRef(t_fileTrans);
+
+    // read Bem
+    MNEBem bemHead(t_fileBem);
+    MNEBemSurface::SPtr bemSurface = MNEBemSurface::SPtr::create(bemHead[0]);
+    MNEProjectToSurface::SPtr mneSurfacePoints = MNEProjectToSurface::SPtr::create(*bemSurface);
+
+    // read digitizer data
+    QList<int> lPickFiducials({FIFFV_POINT_CARDINAL});
+    QList<int> lPickHSP({FIFFV_POINT_CARDINAL,FIFFV_POINT_HPI,FIFFV_POINT_EXTRA,FIFFV_POINT_EEG});
+    FiffDigPointSet digSetSrc = FiffDigPointSet(t_fileDig).pickTypes(lPickFiducials);   // Fiducials Head-Space
+    FiffDigPointSet digSetDst = FiffDigPointSet(t_fileDig).pickTypes(lPickFiducials);
+    digSetDst.applyTransform(transHeadMriRef, false);                                   // Fiducials MRI-Space
+    FiffDigPointSet digSetHsp = FiffDigPointSet(t_fileDig).pickTypes(lPickHSP);         // Head shape points Head-Space
+
+    // Initial Fiducial Alignment
+    // Declare variables
+    Matrix3f matSrc(digSetSrc.size(),3);
+    Matrix3f matDst(digSetDst.size(),3);
+    Matrix4f matTrans;
+    Vector3f vecWeights(digSetSrc.size()); // LPA, Nasion, RPA
+    float fScale;
+
+    // get coordinates
+    for(int i = 0; i< digSetSrc.size(); ++i) {
+        matSrc(i,0) = digSetSrc[i].r[0]; matSrc(i,1) = digSetSrc[i].r[1]; matSrc(i,2) = digSetSrc[i].r[2];
+        matDst(i,0) = digSetDst[i].r[0]; matDst(i,1) = digSetDst[i].r[1]; matDst(i,2) = digSetDst[i].r[2];
+
+        // set standart weights
+        if(digSetSrc[i].ident == FIFFV_POINT_NASION) {
+            vecWeights(i) = 10.0;
+        } else {
+            vecWeights(i) = 1.0;
+        }
+    }
+
+    // align fiducials
+    if(!fitMatched(matSrc,matDst,matTrans,fScale,bScale,vecWeights)) {
+        qWarning() << "Point cloud registration not succesfull.";
+    }
+
+    fiff_int_t iFrom = digSetSrc[0].coord_frame;
+    fiff_int_t iTo = bemSurface->coord_frame;
+    FiffCoordTrans transHeadMri = FiffCoordTrans::make(iFrom, iTo, matTrans);
+
+    // Prepare Icp:
+    VectorXf vecWeightsICP(digSetHsp.size()); // Weigths vector
+    MatrixXf matHsp(digSetHsp.size(),3);
+
+    for(int i = 0; i < digSetHsp.size(); ++i) {
+        matHsp(i,0) = digSetHsp[i].r[0]; matHsp(i,1) = digSetHsp[i].r[1]; matHsp(i,2) = digSetHsp[i].r[2];
+        // set standart weights
+        if((digSetHsp[i].kind == FIFFV_POINT_CARDINAL) && (digSetHsp[i].ident == FIFFV_POINT_NASION)) {
+            vecWeightsICP(i) = 10.0;
+        } else {
+            vecWeightsICP(i) = 1.0;
+        }
+    }
+
+    MatrixXf matHspClean;
+    VectorXi vecTake;
+
+    // discard outliers
+    if(!discardOutliers(mneSurfacePoints, matHsp, transHeadMri, vecTake, matHspClean, fMaxDist)) {
```

Comment:
Maybe it would improve readability if we use the namespace for the three new static functions you implemented. So here it would say: `RTPROCESSINGLIB::discardOutliers`